test/cypress/integration/group-paying.spec.js

```diff
@@ -106,13 +106,9 @@ describe('Group Payments', () => {
 
     // NOTE: TWO HEISENBUGS ARE IN THIS TEST! PLEASE LEAVE THESE COMMENTS FOR FUTURE
     //       REFERENCE IN CASE WE RUN INTO MORE!
-    // NOTE: when we had this command here, Cypress would sometimes fail because cy.window() would
-    //       for some reason return a promise that wouldn't resolve.
```

Comment:
Why was this comment and the one below removed?

Please add it back unless there's a strong reason not to, in which case please help me understand.

Reply:
I remember cypress was failed in this part and moving it back to the above made it work again.
Not sure why failed. But as you suggested, I will run the test again after moving it back to the below and if it doesn't fail, I will revert this change.

Reply:
Having tested several times, moving it back to the below makes Cypress failed.